Doc/tutorial/controlflow.rst

```diff
@@ -36,6 +36,9 @@ to avoid excessive indentation.  An  :keyword:`!if` ... :keyword:`!elif` ...
 :keyword:`!elif` ... sequence is a substitute for the ``switch`` or
 ``case`` statements found in other languages.
 
+If you're making comparison with constants, or checking for specific types or
```

Comment:
1. "making comparison with" isn't a great phrase (I'd use the plural, "comparisons", or say "comparing to")
2. The match statement is really only useful if *the same* variable is used in each condition

```suggestion
If you're comparing the same value to several constants, or checking for specific types or
```